hazelcast/src/main/java/com/hazelcast/internal/networking/nio/AdvancedNetworkStats.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.networking.nio;
+
+import com.hazelcast.instance.ProtocolType;
+import com.hazelcast.internal.json.JsonObject;
+import com.hazelcast.internal.management.JsonSerializable;
+import com.hazelcast.internal.metrics.LongProbeFunction;
+import com.hazelcast.internal.metrics.MetricsRegistry;
+import com.hazelcast.internal.metrics.ProbeLevel;
+import com.hazelcast.spi.annotation.PrivateApi;
+
+import java.util.EnumMap;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static com.hazelcast.util.JsonUtil.getObject;
+
+/**
+ * Stats per {@link ProtocolType} for a single direction of network traffic (inbound or outbound).
+ * <p>
+ * Stores number of bytes sent or received per {@link ProtocolType} depending on whether it is
+ * for inbound or outbound traffic. Works only when Advanced Networking is enabled and
+ * {@link com.hazelcast.config.EndpointConfig} is added for the {@link ProtocolType} in question.
+ */
+@PrivateApi
```

Comment:
You don't need to mark it private. It is in com.hz.internal. So it is private independent of this annotation

Reply:
I didn't know that. Going to remove the annotation.